dodola/cli.py

```diff
@@ -393,6 +400,7 @@ def apply_qplad(
         out_zarr_region=out_zarr_region_d,
         root_attrs_json_file=root_attrs_json_file,
         new_attrs=unpacked_attrs,
+        wet_day_post_correction=wet_day_post_correction,
```

Comment:
```suggestion
        wet_day_post_correction=wetday_post_correction,
```

Curiously enough, the existing `dodola.services.correct_wet_day_frequency` has the "_", so we'll keep that to keep naming consistent in `dodola.services`, I guess